src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -82,7 +83,7 @@ export const CloudEnvironmentModal = ({
   })
 
   const renderDefaultPage = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1 } },
-    _.map(tool => renderToolButtons(tool.label))(filterForTool ? [tools[filterForTool]] : getToolsToDisplay(azureContext))
+    _.map(tool => renderToolButtons(tool.label))(filterForTool ? [tools[filterForTool]] : getToolsToDisplayForCloudProvider(getCloudProviderFromWorkspace(workspace))) //TODO: We should have access to cloudProvider string.
```

Comment:
In the future, I would very much like to be able to remove getCloudProviderFromWorkspace and just have access to the cloudProvider string from the workspace object.

Reply:
It is `cloudPlatform` in the workspace.

Reply:
See other comment - cloudPlatform is not being returned via the workspace details endpoint.